### ISSUE TITLE
Error with helve hammer

### ISSUE THREAD

**Issue by Daniil-Osipenko** [anegostudios/VintageStory-Issues#2595]
### Game Version

v1.18.0-rc.5

### Platform

Windows

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

After creating an ingot or plate with a helve hammer, this hammer does not drop the item. It just stays on the anvil, and when I try to take it, the creation progress resets. I've tried to do something many times.

### How to reproduce

1. Put an iron bloom or ingot on an anvil with a helve hammer. 2. Turn on the hammer. 3. Wait for the creation to finish. 4. The item is not issued, nothing happens.

### Logs

```Log```

**Comment by EnigmaticaGH** [comment]
This may be related to the bug I reported in 1.17: https://github.com/anegostudios/VintageStory-Issues/issues/2177

**Comment by Daniil-Osipenko** [comment]
Yes, it's very similar. But my windmill is without an step-up gear. Forging takes place at a normal speed.

**Comment by McFrugal** [comment]
I've just run into this as well.  In 1.20.3.  The wind is very slow but my helve hammers are working anyway (hitting about once every 2 seconds) and the ingot visually seems to be working but after it gets all the way finished it just doesn't pop off as normal.  If I grab the "ingot" off of the anvil it turns into a half-finished iron bloom.  So, it seems that the progress shown visibly when helve hammers are working is not the real state of the ingot!  I don't know why it's coded that way but it probably shouldn't be...

**Comment by johnsone0455** [comment]
I just ran into this issue as well. The mechanical parts function correctly, as I have a functioning pulverizer. The helve hammer seems to want to function as it goes through the visual animations. The material is placed down on the anvil and the helve hammer begins to function. When the animation appears to be complete and the supposed output is grabbed, what remains in my inventory is a work item that requires the entire process to be repeated. This is in 1.20.4.

**Comment by johnsone0455** [comment]
To update this error, it appears that the issue may be related to the clutch and transmission. After removing both of these pieces, the helve hammer proceeded to function properly!
